packages/config-cli/src/cli/screenshot/screenshot.ts

```diff
@@ -0,0 +1,142 @@
+import { Config, fsa, LogConfig, LogType } from '@basemaps/shared';
+import { mkdir } from 'fs/promises';
+import { Browser, chromium } from 'playwright';
+import { CommandLineAction, CommandLineStringParameter } from '@rushstack/ts-command-line';
+import { z } from 'zod';
+
+enum TileMatrixIdentifier {
+  NZTM = 'NZTM2000Quad',
```

Comment:
Do we not have this as a enum somewhere else?

We should be specific here `Nztm2000Quad` so we know we are talking about nztm2000quad tms and not the other one.